test/WebJobs.Script.Tests.Integration/TestScripts/Proxies/proxies.json

```diff
@@ -178,6 +178,23 @@
       "requestOverrides": {
         "backend.request.headers.accept": "text/plain"
       }
+    },
+    "httpHead": {
+      "matchCondition": {
+        "route": "/headContentLengthTest",
+        "methods": [ "HEAD" ]
+      },
+      "backendUri": "https://httpbin.org/get"
```

Comment:
Can you change the backendUri to something like http://localhost/api/ping? to minimize dependency on external resource?